CHANGELOG.latest.md

```diff
@@ -1,2 +1,10 @@
-- Add canMakePayments method (https://github.com/RevenueCat/react-native-purchases/pull/244)
-- Add PR template (https://github.com/RevenueCat/react-native-purchases/pull/263)
+- This release has some changes that should help prevent "There is no singleton instance" errors.
+    - Added `isConfigured` function to be able to check if the instance of Purchases has been configured before calling any function that accesses the singleton instance.
+    - Some functions that were returning `void`, now return `Promise<void>`: `setAllowSharingStoreAccount`, `setFinishTransactions`, `setSimulatesAskToBuyInSandbox`, `addAttributionData`, `setDebugLogsEnabled`, `syncPurchases`, `setAutomaticAppleSearchAdsAttributionCollection`, `invalidatePurchaserInfoCache`, `presentCodeRedemptionSheet`, and all the subscriber attributes related functions.
```

Comment:
nitpick: `subscriber attributes-related`
Also, will that be clear to customers at first glance? If they're just using stuff for integrations they might miss that they translate into subscriber attributes, right?

Perhaps a table with all of the affected methods organized by type would make things very easy to understand? 

| Integrations | Subscriber attributes | Configuration and other | 
| :-: | :-: | :-: |
| `addAttributionData` | `collectDeviceIdentifiers` | `invalidatePurchaserInfoCache` |
| `setAd` | `setAttributes` |  `presentCodeRedemptionSheet` |
| `setAdGroup` | `setDisplayName` |  `setAllowSharingStoreAccount` |
| `setAdjustID` | `setEmail` |  `setAutomaticAppleSearchAdsAttributionCollection` |
| `setAirshipChannelID` | `setPhoneNumber` |  `setDebugLogsEnabled` |
| `setAppsflyerID` | `setPushToken` |  `setFinishTransactions` |
| `setCampaign` | |   `setSimulatesAskToBuyInSandbox` |
| `setCreative` | |   `syncPurchases` |
| `setFBAnonymousID` | |  |
| `setKeyword` | | |
| `setMediaSource` | | |
| `setMparticleID` | | |
| `setOnesignalID` | | |

Reply:
Wow thanks Andy haha. Much better